AutoMaxLair.py

```diff
@@ -546,7 +551,7 @@ def select_pokemon(inst) -> str:
     inst.push_buttons((b'b', 1.5), (b'b', 1.5))
     
     # Update statistics and reset stored information about the complete run.
-    inst.wins += 1 if inst.num_caught == 4 else 0
+    inst.wins += 1 if inst.lives != 0 else 0
```

Comment:
we can win without catching 4 pokemon (when we're deciding not to catch the legendary in ball saver mode)

Reply:
Nice catch!